platform/darwin/src/MGLFeature.h

```diff
@@ -91,6 +91,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable id)attributeForKey:(NSString *)key;
 
+/**
+ Returns a dictionary that can be serialized as a GeoJSON Feature representation
+ of the `MGLFeature` subclass.
```

Comment:
Pedantry: it’s a representation of an instance of the class, not of the class itself.